contactos/models.py

```diff
@@ -22,8 +21,7 @@ def __unicode__(self):
 class Contact(models.Model):
     """docstring for Contact"""
     contact_type = models.ForeignKey('ContactType')
-    person = models.ForeignKey(Person)
-    popolo_person = models.ForeignKey(PopoloPerson, null=True, blank=True)
+    person = models.ForeignKey(PopoloPerson)
```

Comment:
I think we can import `PopoloPerson` as `Person` now that we've removed the reference to the `popit.models.Person`?